### DISCUSSION TITLE
Font size

### DISCUSSION THREAD

**Discussion by sskit** [lllyasviel/Fooocus#3227] [Q&A]
Hello!
Could you please tell me how I can increase the font in the prompt input field? I've tried changing in the style.css file, but it doesn't help.

**Comment by mashb1t** [comment]
Hey, you can use the id #positive_prompt to address the element in the style.css, then adjust the font size. You miht have to use !important for some overrides if gradio / svelte itself has already defined styles for this element.

**Reply by sskit** [reply]
Thank you for your reply.
I wrote like this, but the font size does not change.....

#positive_prompt {
    padding: 1px;
   font-size: 24px !important;
}

**Reply by mashb1t** [reply]
You can use this snippet:

```
#positive_prompt label textarea {
    font-size: 14px;
}
```

14px is the default, you can increase it however you like. Keep in mind to restart Fooocus after changing the `style.css` file.

**Reply by sskit** [reply]
Thank you very much! It works!
